bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx

```diff
@@ -103,20 +112,34 @@ class ActionsDropdown extends Component {
       isUserModerator,
       meetingIsBreakout,
       hasBreakoutRoom,
+      handleTakePresenter,
     } = this.props;
 
     const {
       pollBtnLabel,
       pollBtnDesc,
       presentationLabel,
       presentationDesc,
+      takePresenter,
+      takePresenterDesc,
     } = intlMessages;
 
     const {
       formatMessage,
     } = intl;
 
     return _.compact([
+      (!isUserPresenter
```

Comment:
Perhaps combine `!isUserPresenter` with `isUserPresenter` right below it -- you can utilize the ternary operator [ that is already there]